packages/sdk/package.json

```diff
@@ -18,7 +18,6 @@
     "require": "./_cjs/src/index.js"
   },
   "scripts": {
-    "deploy": "npm publish --tolerate-republish --access public",
```

Comment:
Just marking this in case we need to roll back changes and deploy a new version.